src/VisualStudio/Core/Def/Implementation/VisualStudioAnalysisScopeService.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.SolutionCrawler;
+using Microsoft.VisualStudio.LanguageServices.Setup;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using IAsyncServiceProvider = Microsoft.VisualStudio.Shell.IAsyncServiceProvider;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation
+{
+    internal sealed class VisualStudioAnalysisScopeService : IAnalysisScopeService
```

Comment:
@mavasani I don't _think_ the OOP solution crawler needs access to this value, but if it does we will also need to create a brokered service to expose this option to OOP.

Reply:
➡️ This code is now removed.